src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/evaluering/EvalueringAktivitetTest.kt

```diff
@@ -0,0 +1,176 @@
+package no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.evaluering
+
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.AktivitetType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.DelvilkårAktivitet
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.DelvilkårVilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.ResultatDelvilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.ResultatVilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.SvarJaNei
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.DelvilkårAktivitetDto
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.evaluering.EvalueringAktivitet.utledResultat
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EnumSource
+
+class EvalueringAktivitetTest {
+
+    @Nested
+    inner class Tiltak {
+
+        @Test
+        fun `hvis ikke lønnet og ikke mottar sykepenger så er resultatet oppfylt`() {
+            val resultat = utledResultat(
+                AktivitetType.TILTAK,
+                delvilkårAktivitetDto(lønnet = SvarJaNei.NEI, mottarSykepenger = SvarJaNei.NEI),
+            )
+            assertThat(resultat.lønnet.resultat).isEqualTo(ResultatDelvilkårperiode.OPPFYLT)
+            assertThat(resultat.lønnet.svar).isEqualTo(SvarJaNei.NEI)
+
+            assertThat(resultat.mottarSykepenger.resultat).isEqualTo(ResultatDelvilkårperiode.OPPFYLT)
+            assertThat(resultat.mottarSykepenger.svar).isEqualTo(SvarJaNei.NEI)
+
+            assertThat(resultat.resultat).isEqualTo(ResultatVilkårperiode.OPPFYLT)
+        }
+
+        @Test
```

Comment:
Kunne disse vært samlet i en nesten class IkkeVurdert? Og evt. legge inn en test med både lønnet og sykepenger ikke vurdert?

Reply:
IKKE_VURDERT på lønnet er kun gyldig for Tiltak, så tenker det er fint å kun ha det under `Tiltak`
Føler egentlige ikke noe behov for å teste at begge er IKKE_VURDERT, men har lagt til det nå